lib/index.es6

```diff
@@ -0,0 +1,34 @@
+import path from "path";
+import fs from "fs";
+
+import {sync as mkdirp} from "mkdirp";
+
+import compile from "./compile";
+import {entries} from "./util";
+
+export default function Interlock (options) {
+  const cwd = process.cwd();
+  options = options || {};
+
+  if (!options.emit || !options.emit.length) {
+    throw new Error("Must define at least one bundle.");
+  }
+
+  // TODO: validate options, or only persist specific values
+  options.context = options.context || cwd;
+  options.outputPath = options.outputPath || path.join(cwd, "dist");
+  options.extensions = options.extensions || [".js", ".jsx", ".es6"];
+  options.ns = options.ns || require(path.join(options.root, "./package.json")).name;
```

Comment:
That may be worth expanding everywhere.  It's not compiled into anything, so there isn't a really great reason not to.